src/Header.js

```diff
@@ -1,10 +1,11 @@
 import React, { Component } from 'react';
 import './Header.css'
+import './App.css';
 
 class Header extends Component {
   render() {
     return (
-      <header>
+      <header className='display-none'>
```

Comment:
and here?

Reply:
Everything has to be hidden other than the splash page for the splash page to be the only visual on the page as the user lands on the page. The display-none class is removed and a block display is added once the user clicks on the see all dogs button.